ucm/sof-skl_hda_card/HiFi.conf

```diff
@@ -103,20 +103,17 @@ SectionDevice."HDMI1" {
 
 	EnableSequence [
 		cdev "hw:sofsklhdacard"
-		cset "name='hif5-0 Jack Switch' on"
-		cset "name='Pin5-Port0 Mux' 1"
+		cset "name='IEC958 Playback Switch' on"
 	]
 
 	DisableSequence [
 		cdev "hw:sofsklhdacard"
-		cset "name='Pin5-Port0 Mux' 0"
-		cset "name='hif5-0 Jack Switch' off"
```

Comment:
What's the reason to remove the enable counterpart?

Reply:
@perexg It seems the IEC958 mute usage is not very systematic and there seems to be no harm in leaving it enabled. It does have to be unmuted as e.g. some systems mute it in their default alsa-ctl init state. It took quite a bit of time e.g. to dig this up in Ubuntu (added in 2005):
“On-board SIS sound card needs muted "IEC958"”
https://bugs.launchpad.net/ubuntu/+source/alsa-utils/+bug/19648

With the older hdac-hdmi driver, the disable sequence was mandatory to avoid problems in routing.